.buildkite/pipeline.yml

```diff
@@ -51,6 +51,31 @@ steps:
                     - BUILDKITE_PULL_REQUEST_BASE_BRANCH=${BUILDKITE_PULL_REQUEST_BASE_BRANCH}
                     - GITHUB_PR_LABELS=${GITHUB_PR_LABELS}
 
+  - label: "Trigger Filebeat"
+    plugins:
+      - monorepo-diff#v1.0.1:
+          diff: "git diff --name-only origin/${GITHUB_PR_TARGET_BRANCH}...HEAD"
+          watch:
+            - path:
+                - filebeat/
+                - .buildkite/filebeat/
+                - .buildkite/scripts
```

Comment:
add `- .buildkite/hooks/` here.

Reply:
saw it just as you where reviewing, pushed 84abcd7